app/js/skrollr.js

```diff
@@ -29,17 +29,30 @@ angular.module("sn.skrollr", [])
          */
         function($window, $document, $rootScope) {
 
-            return {
+            var s,
+                serviceMethods,
+                init = 0;
+
```

Comment:
`init` should probably be a `boolean` and be called `hasBeenInitialised` or something like that. Your variable names and types should be relevant, readable and make sense. If you look at the variable `init` later on the your code, it wouldn't be obvious thats it's a `boolean`, i would have guessed that it's a function.